.rubocop.yml

```diff
@@ -467,3 +467,7 @@ Style/GlobalVars:
   Enabled: true
   Exclude:
     - '**/spec/**'
+
+Style/Alias:
+  Enabled: true
+  EnforcedStyle: prefer_alias_method
```

Comment:
IMO `alias_method` is preferable to `alias` because it creates a method that can be overridden, etc etc

Reply:
👍🏼 👍🏼